File: Data Structures/Linked Lists/Print in Reverse.cpp
/*
   Contributer : github.com/Blind Bluffer
   Email : shivank309@gmail.com
*/
/*
  Print elements of a linked list in reverse order as standard output
  head pointer could be NULL as well for empty list
  Node is defined as 
  struct Node
  {
     int data;
     struct Node *next;
  }
*/
void ReversePrint(Node *head)
{
  // This is a "method-only" submission. 
  // You only need to complete this method.
    if(head==NULL)
       return;
    else
        ReversePrint(head->next);
        cout<<head->data<<endl;
    
}

File: Data Structures/Linked Lists/Print in Reverse.java
/* Contributer : github.com/Shaurya9619
	   Email : shaurya96gairola@gmail.com
*/

void ReversePrint(Node head) {
    if(head==null)
       return;
    else
        ReversePrint(head.next);
        System.out.println(head.data);
    
} 
